### ISSUE TITLE
Read data taken by detector from a database

### ISSUE THREAD

**Issue by dniklaus** [ERNICommunity/dust-measurement-network#2]
1. DB Schema:
 - time series of data
 - detector ID
 - detecor location
 - predictions
 - prediction-actual data
2. Use entity FW for bringing data to app

**Comment by mareklinka** [comment]
As far as I understand it, the influx DB has been decided on, no? If the BE team can live with Influx, I'm fine with it.

@yohny Can you confirm you are expecting Influx?

**Comment by yohny** [comment]
Yes, we expect sensor data to be present in Influx db. From there we will take them to SQL (just a bit more agregated for our purposes) and enhance them with additional properties neeeded for Machine learning.

**Comment by mareklinka** [comment]
Very well, marking the timeseries DB research as closed and reassigning this to the BE team. @yohny please forward this to whoever should be working on this.
